assets/styles/variables/_pages.scss

```diff
@@ -485,10 +485,25 @@ $before-title-title-block-margin-bottom: 0.5em !default;
 /// @type String
 $copyright-float: none !default;
 
+/// Top margin for Copyright on `<p>` elements that are children of elements with an ID of `copyright-page`.
+/// @type String | Map
+/// @since 1.2.0
+$copyright-margin-top: (epub: 2em, prince: 2cm, web: 2em) !default;
```

Comment:
@dannylonglegs This looks like a breaking change; can you confirm? Also it's applied to the `#copyright-page` element rather than paragraph children of that element. I'm a bit confused!